app/code/Magento/Checkout/Block/Checkout/AttributeMerger.php

```diff
@@ -388,4 +387,36 @@ protected function orderCountryOptions(array $countryOptions)
         }
         return array_merge($headOptions, $tailOptions);
     }
+
+    /**
+     * @return null|string
+     */
+    protected function getDefaultPrefix()
```

Comment:
Because we do not encourage inheritance-based API. So composition in favor of inheritance. It's easier to maintain and reuse.